README.md

```diff
@@ -106,6 +107,17 @@ The Akita is a Japanese dog breed known for its loyalty and dedication. It's als
 
 We were also very inspired by the story of [Hachikō](https://en.wikipedia.org/wiki/Hachik%C5%8D) - a dog of immense loyalty. Hachikō was an Akita who would meet with his owner every day at the train station. Even after his owner passed away, Hachikō continued to wait for his owner at the train station for more than 9 years. Hachikō continues to have a legacy in Japanese culture, with various statues created in his memory - there was even an [American film](https://en.wikipedia.org/wiki/Hachi:_A_Dog%27s_Tale) based on him.
 
+### Akita Data
+Akita collects data on
+
+- your time spent and number of visits on all websites,
+- the Payment Pointers present on the websites you visit,
+- and the currency and amount of payments streamed to [Payment Pointers](https://paymentpointers.org/) via Web Monetization.
+
+You can view an example of the data collected by Akita in [`examples/example_data.json`](examples/example_data.json).
+The data collected by Akita is stored on your machine, in your browser's local storage.
```

Comment:
May be good to explicitly say that it is not uploaded anywhere else (we don't back up the data for them).

The data stays in their hands by design, consequently though it's not backed up anywhere by us. So if they delete it, it's gone forever.